libs/expo/betterangels/src/lib/ui-components/Attachments/ImageComponent.tsx

```diff
@@ -1,30 +1,46 @@
 import { useMutation } from '@apollo/client';
 import { PlusIcon } from '@monorepo/expo/shared/icons';
 import { Colors, Radiuses, Spacings } from '@monorepo/expo/shared/static';
-import { IconButton } from '@monorepo/expo/shared/ui-components';
+import {
+  CircularLoading,
+  IconButton,
+} from '@monorepo/expo/shared/ui-components';
+import { Dispatch, SetStateAction } from 'react';
 import { Image, View } from 'react-native';
 import { DELETE_NOTE_ATTACHMENT } from '../../apollo';
 import {
   DeleteNoteAttachmentMutation,
   DeleteNoteAttachmentMutationVariables,
 } from '../../apollo/graphql/__generated__/mutations.generated';
 
+interface IImage {
```

Comment:
just curious, does this have to be defined in each component? There's no way to reuse it?

Reply:
if we create types file and import from there.

Reply:
I'm not as familiar with fe convention, so I'm not sure if/when we'd want to do that. Do you think it makes sense to have a types file now, or can we hold off on that? May be a good idea to create a ticket for it just so it's in the backlog and we know it's something we eventually want to do